lib/ansible/modules/user.py

```diff
@@ -529,6 +536,7 @@ def __init__(self, module):
         self.role = module.params['role']
         self.password_expire_max = module.params['password_expire_max']
         self.password_expire_min = module.params['password_expire_min']
+        self.umask = module.params['umask']
```

Comment:
It looks like this won't work with `luseradd`, just `useradd`. We should check this option isn't provided with the `local` option. Looks good to merge after that :+1:

Reply:
@s-hertel I've set this to fail when both options have values.
Is this needs to change to warn that `umask` will not be set and check whenever `umask` has been set when both options have values, please tell me to change this.